src/PrestaShopBundle/Controller/Admin/Configure/ShopParameters/SearchController.php

```diff
@@ -65,4 +70,46 @@ public function indexAction(Request $request, AliasFilters $filters): Response
             ],
         ]);
     }
+
+    /**
+     * @AdminSecurity("is_granted('read', request.get('_legacy_controller'))")
+     *
+     * @param Request $request
+     *
+     * @return JsonResponse
+     */
+    public function searchAliasesForAssociationAction(Request $request): JsonResponse
+    {
+        try {
+            /** @var AliasForAssociation[] $searchTerms */
+            $searchTerms = $this->getQueryBus()->handle(new SearchAliasesForAssociation(
+                $request->get('query', ''),
+                (int) $request->get('limit', 20)
```

Comment:
put 20 into constant as DEFAULT_ALIAS_SEARCH_LIMIT (can even be here in controller, or inside the command I think)